api/blocks/blocks_test.go

```diff
@@ -176,7 +176,7 @@ func initBlockServer(t *testing.T) {
 	addr := thor.BytesToAddress([]byte("to"))
 	cla := tx.NewClause(&addr).WithValue(big.NewInt(10000))
 	trx := tx.MustSign(
-		new(tx.Builder).
+		new(tx.LegacyBuilder).
```

Comment:
I suppose we should add new txs to rest of the old tests to keep em compatible

Reply:
sure, we can have the two typed txs in the init server function